catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/FilterToTextDelegate.java

```diff
@@ -20,6 +20,22 @@
 
 public class FilterToTextDelegate extends FilterDelegate<String> {
 
+    // Custom functions
+    @Override
+    public String propertyIsEqualTo(String functionName, List<Object> arguments, Object literal) {
+        switch (functionName) {
+        case "divisibleBy":
+            return propertyIsDivisibleBy((String) arguments.get(0), (Long) arguments.get(1)) + '='
+                    + literal;
+        case "proximity":
+            return propertyIsInProximityTo((String) arguments.get(0),
```

Comment:
changed GeotoolsFilterAdapter to throw an exception if it couldn't find a convert to convert the types